--- xfce-mcs-manager-4.2.2/configure.ac.orig	2005-05-15 10:06:34.000000000 +0200
+++ xfce-mcs-manager-4.2.2/configure.ac	2005-05-19 01:54:36.686368600 +0200
@@ -48,7 +48,7 @@
 dnl Check for i18n support
 BM_I18N([xfce-mcs-manager], [ar az be bg bn_IN ca cs da de el en_GB eo es es_MX
                              et eu fa fi fr gu he hi hu it ja ko lt mr ms
-			     nb_NO nl pa pt_BR pt_PT pl ro ru sk sv ta
+			     nb nl pa pt_BR pt pl ro ru sk sv ta
 			     tr uk vi zh_CN zh_TW])
 
 dnl Check for libxfcegui4
