--- xfce-mcs-manager-4.2.3/configure.ac~	2005-11-09 06:09:14.820100592 +0100
+++ xfce-mcs-manager-4.2.3/configure.ac	2005-11-09 06:09:53.541214088 +0100
@@ -48,7 +48,7 @@
 dnl Check for i18n support
 BM_I18N([xfce-mcs-manager], [af ar az be bg bn_IN ca cs da de el en_GB eo
 			     es es_MX et eu fa fi fr gu he hi hu it ja ko
-			     ku lt mr ms nb_NO nl pa pt_BR pt_PT pl ro ru
+			     ku lt mr ms nb nl pa pt_BR pt pl ro ru
 			     sk sl sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for libxfcegui4
