--- xfce-mcs-manager-4.2.1/configure.ac	2005-03-12 15:19:04.000000000 +0100
+++ fix/xfce-mcs-manager-4.2.1/configure.ac	2005-03-16 02:40:59.802622920 +0100
@@ -47,8 +47,8 @@
 
 dnl Check for i18n support
 BM_I18N([xfce-mcs-manager], [ar az be bg ca cs da de en_GB es es_MX et eu fa \
-                             fi fr gu he hi hu it ja ko lt ms nb_NO nl pa \
-                             pt_BR pt_PT pl ro ru sk sv ta tr uk vi zh_CN \
+                             fi fr gu he hi hu it ja ko lt ms nb nl pa \
+                             pt_BR pt pl ro ru sk sv ta tr uk vi zh_CN \
                              zh_TW])
 
 dnl Check for libxfcegui4
