diff -urN xfce-mcs-manager-4.3.99.1/configure.in xfce-mcs-manager-4.3.99.1.new/configure.in
--- xfce-mcs-manager-4.3.99.1/configure.in	2006-09-03 13:46:45.000000000 +0200
+++ xfce-mcs-manager-4.3.99.1.new/configure.in	2006-09-04 15:06:18.000000000 +0200
@@ -76,7 +76,7 @@
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mr ms nb_NO nl pa pt_BR pt_PT pl ro ru sk sv ta tr uk vi zh_CN zh_TW])
+XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mr ms nb nl pa pt_BR pt pl ro ru sk sv ta tr uk vi zh_CN zh_TW])
 
 dnl ***********************************
 dnl *** Check for required packages ***
